tools/oeedger8r/Emitter.ml

```diff
@@ -655,15 +663,15 @@ let gen_enclave_code (ec : enclave_content) (ep : edger8r_params) =
            in
            flatten_map
              (gen_serialize (arg ^ gen_c_deref param_count) param_count)
-             (get_deepcopy (get_param_atype ptype) structs ep)) ]
+             (get_deepcopy (get_param_atype ptype))) ]
         |> List.flatten
       in
       let params =
         flatten_map (gen_serialize "" "1")
           (List.filter (fun (p, _) -> is_in_ptr p || is_inout_ptr p) plist)
       in
       (* Note that the indentation for the first line is applied by the
-     parent function. *)
+         parent function. *)
```

Comment:
Oof, `ocamlformat` doesn't fix comment indentations, but I accidentally triggered an auto-fill which went and fixed all the moved comments. Just gonna leave it.